test/prober/prober.go

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2020 The Knative Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package prober
+
+import (
+	"go.uber.org/zap"
+	"knative.dev/eventing/test/lib"
+	"testing"
+	"time"
+)
+
+var (
+	Version = "v0.7.0"
```

Comment:
Hmm, where is this `Version` being used? Can we have a comment here?

Reply:
I'll add a comment.